non-normative/smbios.adoc

```diff
@@ -1,4 +1,4 @@
-Note the DMTF requirements on the 64-bit SMBIOS 3.0 entry point (cite:[SMBIOS] Section 5.2.2) and data structures (cite:[SMBIOS] Section 6.2).
+Note the DMTF requirements on the 64-bit SMBIOS 3.0 entry point outlined in Section 5.2.2, as well as the conformance guidelines provided in Annex A of the cite:[SMBIOS] specification.
```

Comment:
Note the DMTF requirements on the 64-bit SMBIOS 3.0 entry point (cite:[SMBIOS] Section 5.2.2) and the conformance guidelines (cite:[SMBIOS] Annex A).

The way you have it now suggests Section 5.2.2 is something in this document, while "of the cite:[SMBIOS] specification" might get rendered weird (of the [22] specification or similar).

Reply:
Noted, will make the necessary changes.